### DISCUSSION TITLE
Compatible python package

### DISCUSSION THREAD

**Discussion by Fischchen** [jchristn/SuperSimpleTcp#71] [General]
Is there any compatible python package or is there any encryption I have to think of when using the build in python package to communicate with this?

**Comment by jchristn** [comment]
Hi @Fischchen SimpleTcp doesn't use any kind of framing mechanism, so it should be compatible with any other TCP library out there (that also doesn't use its own framing).  If you enable SSL in SimpleTcp, you'll need to use SSL on the other side, and set the parameters around mutual authentication and acceptance of invalid certificates as needed.
